WISE-Community/WISE-Client
# refactor(Peer Chat): Refactor into smaller components and update UI

## Changes
- Extract previous work display to its own component
- Extract question bank to its own component
- Extract chat box to its own component 
- Update styles and make responsive
- Show student names

Closes #409.

## Test
- Make sure Peer Chat components work as before
- Verify that workgroup usernames are correctly shown for previous work and chat message displays
